src/common/low_precision_transformations/src/eliminate_fake_quantize.cpp

```diff
@@ -46,25 +47,63 @@ bool EliminateFakeQuantizeTransformation::transform(TransformationContext& conte
 }
 
 namespace {
-bool check_interval(const std::shared_ptr<opset1::Constant>& constant, const float value) noexcept {
+bool check_interval(const std::shared_ptr<opset1::FakeQuantize>& fq,
+                    const std::shared_ptr<opset1::Constant>& constant,
+                    const float value,
+                    const float max_diff,
+                    const bool exact_comparison) noexcept {
+    bool need_to_check_intervals = false;
     const auto& constant_values = constant->cast_vector<float>();
     for (const auto constant_value : constant_values) {
         if (std::fabs(constant_value - value) > std::numeric_limits<float>::epsilon()) {
+            const auto diff = std::fabs(constant_value - value);
+            if ((exact_comparison && (std::fabs(constant_value - value) > std::numeric_limits<float>::epsilon())) ||
```

Comment:
Please apply the changes from my [previous comment](https://github.com/openvinotoolkit/openvino/pull/17054#discussion_r1172323350)

Reply:
discussed offline, will be fixed in separate PR